message/injector.cpp

```diff
@@ -1581,6 +1581,13 @@ void Injector::insertMessages()
 
         // Now we insert the headers and bodies of every MIME bodypart.
 
+        Bodypart *bp;
+        if ( m->hasPGPsignedPart() ) {
+            EString pnr( "raw-pgp-signed" );
```

Comment:
I don't understand the point of this; why not use the parent instead (the one with c-t multipart/signed)?